app/models/visit_group.rb

```diff
@@ -35,7 +35,6 @@ class VisitGroup < ActiveRecord::Base
   attr_accessible :window_before
   attr_accessible :window_after
   acts_as_list scope: :arm
-  scope :at_position, ->(position) { where(position: position) }
```

Comment:
I meant to remove this on my previous story / branch (PR #278) which was already merged in.  It is not used anywhere (see first two commit messages)...   I probably should have done this straight to master branch.